ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -27,7 +27,7 @@
     public static final By VISIT_LINK = By.className("toast-item-wrapper");
     public static final By SELECT_LOCATION = By.id("w5");
     public static final By FORM_EXIST = By.className("action-section");
-    private static final By START_VISIT = By.id("org.openmrs.module.coreapps.createVisit");
+    private static final By START_VISIT = By.linkText("Start Visit");
```

Comment:
any need for changing this element?

Reply:
Yes! The element fails to be retrieved only with chrome headless=true. Retrieving the element using By.linkText() makes the test to pass on all instances. cc: @kaweesi

Reply:
The test fails at that point due to failure to retrieving the element by the set attribute. By.linkTex() makes it unique to be retrieved by selenium API